packages/babel-plugin-transform-flow-strip-types/test/fixtures/strip-types/strip-iterator/input.js

```diff
@@ -0,0 +1,15 @@
+declare class A {
```

Comment:
It looks like the output for this file is missing?

Reply:
That's expected (i.e. there is no output)

Reply:
@existentialism Thanks!!